src/patternfly/components/Select/select-checkbox-groups.hbs

```diff
@@ -0,0 +1,49 @@
+<div class="pf-c-select__menu{{#if select-multi--modifier}} {{select-multi--modifier}}{{/if}}"
+  {{#unless select--IsExpanded}}hidden{{/unless}}
+  {{#if select-multi--attribute}}
+    {{{select-multi-attribute}}}
+  {{/if}}>
+  <form class="pf-c-form">
+  <fieldset class="pf-c-form__fieldset" aria-labelledby="{{id}}-label">
+    <div class="pf-c-select__menu-group">
+      <div class="pf-c-select__menu-group-title">Status</div>
+      {{#> check check--modifier="pf-c-select__menu-item"}}
+        {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-running" name="running"')}}{{/check-input}}
```

Comment:
Is it weird that when you hover  the row in the select menu that the background image changes, but only the checkbox and `label` are clickable? I was expecting the whole row to be clickable, though not sure how that would work exactly :)

Reply:
I'm going to defer to JS here since this is behavior.